### ISSUE TITLE
AttributeError: function 'cb_GetMajor' not found

### ISSUE THREAD

**Issue by Infernio** [wrye-bash/wrye-bash#457]
When launching Oblivion, someone was hit with this traceback:

```
 Traceback (most recent call last):
  File "bash\bash.pyo", line 187, in main
  File "bash\bash.pyo", line 281, in _main
  File "zipextimporter.pyo", line 82, in load_module
  File "bash\bosh\__init__.pyo", line 45, in <module>
  File "zipextimporter.pyo", line 82, in load_module
  File "bash\bosh\_mergeability.pyo", line 27, in <module>
  File "zipextimporter.pyo", line 82, in load_module
  File "bash\cint.pyo", line 214, in <module>
  File "ctypes\__init__.pyo", line 379, in __getattr__
  File "ctypes\__init__.pyo", line 384, in __getitem__
AttributeError: function 'cb_GetMajor' not found
```

CBash-related obviously, so bad news already.
Being reported on AFKMods thread right now, has apparently also hit people in other random places before:
- https://www.afkmods.com/index.php?/topic/4966-wrye-bash-all-games/page/78/&tab=comments#comment-175692
- https://www.reddit.com/r/oblivion/comments/8wtyxj/wrye_bash_mod_help/
- https://forums.nexusmods.com/index.php?/topic/6793302-wrye-bash-crashing-immediately/

I've hit this with my own setup once as well, but only when launching via PyCharm.
Went away again after a while, and I can't reproduce it now. Opening the issue here to let @Utumno know.

**Comment by Infernio** [comment]
Actually, I have an idea of what it might be. Asking the person again.

**Comment by Infernio** [comment]
Oops, forgot to put a space in there 😅

**Comment by Utumno** [comment]
All in the day's work :P

Great, btw we should remove print statements entirely ;)

**Comment by Infernio** [comment]
Nevermind, that commit doesn't actually fix it. Should still be included since it's a friendlier error message for people who are using an incompatible version of CBash.dll.

**Comment by Infernio** [comment]
So, the user tried the new version and it did not stop the crash and log into the bugdump, which means that they have CBash 0.7, but somehow ctypes loading of the functions is failing. Bugdump is entirely empty, which makes sense since that error occurs early during boot.

**Comment by Infernio** [comment]
Closing all CBash issues as `R-wont-fix` (#530).
